src/app/components/ItemFilters/ItemFilter.jsx

```diff
@@ -122,17 +121,7 @@ const ItemFilter = ({
   const open = mobile ? mobileIsOpen : isOpen;
 
   return (
-    <FocusTrap
-      focusTrapOptions={{
-        clickOutsideDeactivates: true,
-        onDeactivate: () => {
-          if (!mobile) manageFilterDisplay('none');
-        },
-        returnFocusOnDeactivate: false,
-      }}
-      active={isOpen}
-      className="item-filter"
-    >
+    <div className="item-filter">
```

Comment:
does this have any accessibility implications? ie is focus going to be affected?

Reply:
Yes, but this page will soon be replaced so I'm not too concerned.